guides/back-end-projects.md

```diff
@@ -0,0 +1,104 @@
+---
+explains: The style guide to the way we organize our back-end projects
+---
+
+# How we use docker and docker-compose
+
+A lot of the software we make at City of Amsterdam uses docker containers.
```

Comment:
"at **the** City of Amsterdam"  but i think the "for the municipality of Amsterdam" is even better because that covers the institution we work for. We are not just any company that resides in the city.

Reply:
The official English name of the organisation is actually the City of Amsterdam, so this is correct.

https://www.amsterdam.nl/bestuur-organisatie/bedrijfsvoering/stijlweb/bouwstenen/logo-gemeente/

Reply:
> The official English name of the organisation is actually the City of Amsterdam, so this is correct.

Ok, i stand corrected. That just leaves the missing `the`.